test/models/tag_color_manipulator_test.rb

```diff
@@ -0,0 +1,12 @@
+require "test_helper"
+
+class TagColorManipulatorTest < ActiveSupport::TestCase
```

Comment:
Is this the type of test you wanted for the PORO

Reply:
This is looking great! And the integration test looks perfect.

I just want to take it one step farther. I don't want _any_ logic in the initializer. Essentially, I want that entire file to read... (Feel free to mess with the PORO name here.)

```ruby
ActionView::Base.field_error_proc = proc do |html_tag, instance|
  FieldErrorTagBuilder.new(html_tag, instance).error_field
end
```

Once all of the logic is in the PORO we can test everything, including the tag index logic. It might also make it a bit easier to read through because the tests call out exactly what is going on.